include/deal.II/particles/particle_handler.h

```diff
@@ -208,7 +208,9 @@ namespace Particles
     /**
      * Return the number of particles that live on the given cell.
      *
-     * @note While this function is used in step-19, it is not an efficient
+     * @note
+     *  //TODO this not true now no?
```

Comment:
Yes, this is no longer correct. Please remove the whole note.

Reply:
Perfect. I think we will need to adapt the discussion in step-19 and step-19 future extension to this.